johnzon-mapper/src/main/java/org/apache/johnzon/mapper/converter/ZonedDateTimeConverter.java

```diff
@@ -0,0 +1,202 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.johnzon.mapper.converter;
+
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import static org.apache.johnzon.mapper.converter.Java8Converter.ZONE_ID_UTC;
+
+public class ZonedDateTimeConverter extends Java8Converter<ZonedDateTime> {
```

Comment:
@bcluap -- check this out: https://github.com/derekm/jersey/blob/fix-httpdateformat/core-common/src/main/java/org/glassfish/jersey/message/internal/HttpDateFormat.java#L62

https://github.com/derekm/jersey/blob/fix-httpdateformat/core-common/src/test/java/org/glassfish/jersey/message/internal/HttpDateFormatTest.java

I came up with the algorithm in readDate() and the tests, if you want to use them!